### ISSUE TITLE
Ansible playbooks should be update for 0.8.0

### ISSUE THREAD

**Issue by ymesika** [istio/istio#5985]
As reported in istio/issues#368 v0.8.0 ansible playbooks are not correctly updated for changes in this release.

All kuberentes `istio*.yaml` files are now `istio-demo*.yaml` files.
We should either:
1. Update the playbook to match the new names
2. Use Helm too when installing Istio using Ansible (preferred)

**Comment by cmoulliard** [comment]
Additional remark. Prometheus has been removed from the install - distro

```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Path /home/sblanc/.istio/istio-0.8.0/install/kubernetes/addons/prometheus.yaml does not exist !", "rc": 257}

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "oc create -f ~/.istio/istio-0.8.0/install/kubernetes/istio.yaml", "delta": "0:00:00.130442", "end": "2018-06-05 13:58:28.770823", "msg": "non-zero return code", "rc": 1, "start": "2018-06-05 13:58:28.640381", "stderr": "error: the path \"/home/sblanc/.istio/istio-0.8.0/install/kubernetes/istio.yaml\" does not exist", "stderr_lines": ["error: the path \"/home/sblanc/.istio/istio-0.8.0/install/kubernetes/istio.yaml\" does not exist"], "stdout": "", "stdout_lines": []}
```

**Comment by geoand** [comment]
@sdake I can obviously  fix this easily, but I think we first need to figure out if the Ansible scripts are going to try and support all versions (like they do ATM), or only support the latest version

**Comment by sdake** [comment]
@geoand I'd suggest just supporting only the latest release (and we backport your fix to the 0.8 branch and release it in 0.8.1).  No sense supporting 0.2/0.3/0.4/etc - it will make your life dreadful :)

**Comment by geoand** [comment]
@sdake Makes sense. I will fix the issue in a couple days max. 

Do we have an ETA for `0.8.1`? If we are going to only support the latest version then PR #4623 should also be included in `0.8.1`

**Comment by geoand** [comment]
@sdake I also opened #6144 to take care of the various updates needed
